Sources/SwiftFormat/Rules/OmitExplicitReturns.swift

```diff
@@ -60,7 +60,7 @@ public final class OmitExplicitReturns: SyntaxFormatRule {
     guard let accessorBlock = binding.accessorBlock,
       let transformed = transformAccessorBlock(accessorBlock)
     else {
-      return node
+      return super.visit(node)
```

Comment:
If the condition was not met, the `node` was returned immediately, preventing the additional `visit` logic from executing.  
By calling `super.visit`, the `visit` process continues, allowing the formatting logic in `visit(_ node: ClosureExprSyntax)` to be applied.